Application/EdFi.Ods.Api/Security/Authorization/Repositories/AggregateRootCriteriaProviderDecoratorBase.cs

```diff
@@ -31,31 +30,17 @@ public abstract class AggregateRootCriteriaProviderAuthorizationDecoratorBase<TE
         private readonly IAggregateRootCriteriaProvider<TEntity> _decoratedInstance;
         private readonly IAuthorizationFilterContextProvider _authorizationFilterContextProvider;
         private readonly IAuthorizationFilterDefinitionProvider _authorizationFilterDefinitionProvider;
-        private readonly IEducationOrganizationIdNamesProvider _educationOrganizationIdNamesProvider;
-        private readonly Lazy<List<string>> _sortedEducationOrganizationIdNames;
 
         private readonly ILog _logger;
```

Comment:
`_logger` is assigned but never used, can we remove it?

Reply:
Done.